### ISSUE TITLE
logging into Discourse with GitHub

### ISSUE THREAD

**Issue by brainwane** [pypa/packaging-problems#363]
In the  [decommissioned](https://groups.google.com/forum/#!msg/pypa-dev/rUNsfIbruHM/LCEx-CB5AgAJ) mailing list
it links to the reason for [close](https://groups.google.com/forum/#!msg/pypa-dev/twf9HCGfv3k/t2HJwzF-AgAJ)

>3. The Packaging category on Python's Discourse forum
https://discuss.python.org/c/packaging , which started about a year and
a half ago[3]. Very wide scope. You can log in with an account, or with
Facebook or GitHub or via email. About 21 posts per month. Hosted by PSF
with an open source application that's under active development.

Can some please explain how to login by github is done.
I want to post to this list but don't want The mailing list sent to my email.

_Originally posted by @frog-o in https://github.com/pypa/packaging.python.org/issues/732#issuecomment-636044984_

**Comment by brainwane** [comment]
@frog-o, does https://discuss.python.org/faq answer your question?

**Comment by frog-o** [comment]
@brainwane  for some reason i was not notified or i missed this when you move my post.   So that is the reason for such a late reply plus I only look at this stuff on the weekend if i have it off.   

I have a hard time with comprehension and communication so can you tell me: 

What I am i supposed to see ?
Why is this called a faq I don't see any question-marks or question in it at all? 

I recognize now my question was a little vague and in truth is was not meant to be a question but a statement saying it not easy to use the mailing list and i want it to be seen by the author. 

some reason mailing list are hard  

* subscription based 
* very hard to read  (i read many there is no formatting, sometimes previouse commit left other times not.   Sometimes i have to look for next button or some other weird labile)it is a pain  
* require some form of passwords( I think i spent 6 hours on "I an not a robot page" trying to create a account) no fun   
-or *ask for more permission (I tried to just click the login button and it did give me login with GitHub option and when i click on that is ask GitHub to give you guys permission to my email.)
* expect you to read another licence agreement. 

I was also trying to ask the person who work on [issue 732](https://github.com/pypa/packaging.python.org/issues/732#issuecomment-636044984) to also work on a guide on how to use mailing list since there related.  Know that i think about it; This should of been a new issue and I probably should of mention the author.     

I am new to all of this and even if i made a new issue this would have not been appropriate place since it does not deal with packaging. 

**This is very frustrating**. 

In summary 
I don't want to subscribe and it is very vague weather it is going to subscribe you or not.
I really don't want to open up permission on my GitHub account.    
Mailing list I feel are a pain for me because of my disability.

**sorry i don't know where to put this** it said i should use the mailing list or IRC for comments such as this but I am unable.

**Comment by merwok** [comment]
A partial answer about the login with Github: like all delegated login systems, this does not give access to your Github account, nor to your email account, it only communicates your email address (which is necessary to identify you and your messages)

**Comment by merwok** [comment]
> I want to post to this list but don't want The mailing list sent to my email.

Like all forums, python discuss uses your email address as an identifer, then you post and read messages in your web browser. There is a setting to send messages by email too, but that’s entirely opt-in.
